components/PostCardList/PostCardListItem.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import Post from "../../models/Post";
+import calculateFinalPot from "../../utilities/calculateFinalPot";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+import getRelativeDateString from "../../utilities/getRelativeDateString";
+import getSIMetricPrefixData from "../../utilities/getSIMetricPrefixData";
+import Card from "../Card";
+import { ThumbsUpIcon } from "../Icon";
+import PlayingCard from "../PlayingCard";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  post: Post;
+  onClick?: React.MouseEventHandler<HTMLElement>;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItem({ post, ...props }: Props) {
+  const { heroIndex, playerCards } = post.gameSituation;
+  const { left, right } = playerCards[heroIndex]!;
+  const postType = React.useContext(PostTypeContext);
+  const [date, mobileDateUnit, dateUnit] = getRelativeDateString(
+    postType === PostType.recent ? post.createdAt : post.lastUpdatedAt
+  );
+
+  return (
+    <Root {...props}>
+      <PlayerHandArea>
+        <PlayerHandAreaBackGround />
+        <HandCard suit={left.suit} rank={left.rank} />
+        <HandCard suit={right.suit} rank={right.rank} />
+      </PlayerHandArea>
+
+      <PostTitle>{post.title}</PostTitle>
+
+      <LikeArea>
+        <ThumbsUpIcon />
+        {post.likes}
+      </LikeArea>
+
+      <Attributes>
+        <Attribute>
+          <h5>Play at</h5>
+          <span>
+            {getPositionByPlayerAndIndex(
+              post.gameSituation.playerLength,
+              post.gameSituation.heroIndex
+            )}
+          </span>
+        </Attribute>
+
+        <Attribute>
+          <h5>Ended at</h5>
+          <span>
+            {post.gameSituation.river
+              ? EndedStreet.river
+              : post.gameSituation.turn
+              ? EndedStreet.turn
+              : post.gameSituation.flop
+              ? EndedStreet.flop
+              : EndedStreet.preflop}
+          </span>
+        </Attribute>
+
+        <Attribute>
+          <h5>Final Pod</h5>
+          <span>{`${getSIMetricPrefixData(
+            calculateFinalPot(post.gameSituation)
+          )} BB`}</span>
+        </Attribute>
+
+        <Attribute>
+          <h5>{postType === PostType.recent ? "Posted" : "Last Update"}</h5>
+          <span>
+            {date}
+            <DateUnitInMobile>{mobileDateUnit}</DateUnitInMobile>
+            <DateUnit>{dateUnit}</DateUnit>
+            &nbsp;ago
+          </span>
+        </Attribute>
+      </Attributes>
+    </Root>
+  );
+}
+
+const Root = styled(Card)`
+  display: grid;
+  grid-template-columns: minmax(15%, 100px) 1fr;
+  grid-template-areas:
+    "playinghand-area title-area"
```

Comment:
Use proper terms. Please.

If you are not sure, refer to dictionaries or [google](https://www.google.com/search?q=poker+playing+hand).